From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tom Zanussi <zanussi@kernel.org>
Date: Mon, 2 Dec 2019 13:30:02 -0600
Subject: [PATCH] Linux 4.14.164-rt73 REBASE

Signed-off-by: Tom Zanussi <zanussi@kernel.org>
---
 localversion-rt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/localversion-rt b/localversion-rt
index 90290c642ed5..e8ada8cdb471 100644
--- a/localversion-rt
+++ b/localversion-rt
@@ -1 +1 @@
--rt29
+-rt73
-- 
https://clearlinux.org

