From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Tue, 28 Jan 2020 11:35:14 -0600
Subject: [PATCH] fix drm/i915: drop duplicate eb_use_cmdparser function

---
 drivers/gpu/drm/i915/i915_gem_execbuffer.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/gpu/drm/i915/i915_gem_execbuffer.c b/drivers/gpu/drm/i915/i915_gem_execbuffer.c
index f0907daaa..7bd920684 100644
--- a/drivers/gpu/drm/i915/i915_gem_execbuffer.c
+++ b/drivers/gpu/drm/i915/i915_gem_execbuffer.c
@@ -275,11 +275,6 @@ static inline bool eb_use_cmdparser(const struct i915_execbuffer *eb)
 		 eb->args->batch_len);
 }
 
-static inline bool eb_use_cmdparser(const struct i915_execbuffer *eb)
-{
-	return eb->engine->needs_cmd_parser && eb->batch_len;
-}
-
 static int eb_create(struct i915_execbuffer *eb)
 {
 	if (!(eb->args->flags & I915_EXEC_HANDLE_LUT)) {
-- 
https://clearlinux.org

